index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>your profile</title>
+    <link rel="stylesheet" href="src/style/style.css">
+    <link rel="stylesheet" href="src/style/reset.css">
+</head>
+
+<body>
+    <nav id="top-menu">
+        <ul class="profile-menu-wrap">
+            <li>
+                <a class="menu-move">
+                    <img src="src/images/icon/icon-arrow-left.png" alt="페이지 뒤로가기 이미지">
+                </a>
+            </li>
+            <li>
+                <a class="menu-move">
+                    <img src="src/images/icon/icon-more-vertical.png" alt="로그인 설정 및 로그아웃 메뉴 이동">
+                </a>
+            </li>
+        </ul>
+    </nav>
+
+    <section class="user-info">
+        <div class="user-follow-number">
+            <div class="followers-wrap">
+                <p class="followers-num">2950</p>
+                <p clsss="followers-txt">followers</p>
+            </div>
+
+        <img src="src/images/basic-profile-img.png" alt="사용자 프로필 이미지">
+
+            <div class="followings-wrap">
+                <p class="followings-num">128</p>
+                <p clsss="followings-txt">followers</p>
+            </div>
+        </div>
+        
+
+        <h1 class="user-profile-name">애월읍 위니브 감귤농장</h1>
+        <p casss="user-profile-id">@weniv_Mandarin</p>
+        <p class="user-profile-info">애월읍 감귤 전국 배송, 귤따기 체험, 감귤 농장</p>
+
+        <img src="src/images/icon/icon-message-circle.png" alt="현재 프로필 사용자에게 메시지 전송하기">
+        <!-- <img src="/src/" alt="사용자 팔로우"> -->
+        <img src="src/images/icon/icon-share.png" alt="현재 프로필을 SNS에 공유하기">
+    </section>
+
+    <section class="product">
+        <h2 class="product-sale">판매 중인 상품</h2>
+
+        <article class="product-item">
+            <img src="src/images/product-img-example.png" alt="상품: 애월읍 노지 감귤">
+            <p class="product-txt">애월읍 노지 감귤</p>
+            <p class="product-price">35,000원</p>
+        </article>
```

Comment:
클릭을 했을 때 해당 상품 페이지로 이동이 되어야하는 걸 고려해서 article 태그 하위에 있는 자식 요소들을 article 대신 a 태그로 감싸는건 어떨까요?
그리고 상품들이 여러개 등록이 될 것을 고려했을때 li 태그를 사용해보는 건 어떨까요?
```suggestion
<li class="product-item">
    <a href="none">
        <img src="src/images/product-img-example.png" alt="상품: 애월읍 노지 감귤">
        <p class="product-txt">애월읍 노지 감귤</p>
        <p class="product-price">35,000원</p>
    </a>
</li>
```